src/styles/sidebar/components/top-bar.scss

```diff
@@ -77,37 +69,16 @@
   &:hover {
     color: $gray-dark;
   }
-  &--active {
+
+  &.is-active {
     color: $gray-dark;
   }
-}
 
-// Button which indicates that other users have made or edited annotations
-// on the current page and applies them when clicked
-//
-// The button displays a 'refresh' icon and an update count, both of which
-// should be vertically centered within the icon.
-.top-bar__apply-update-btn {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  flex-shrink: 0;
-  margin-right: 5px;
+  &--refresh {
+    color: $brand;
 
-  border-radius: 2px;
-  color: white;
-  cursor: pointer;
-  font-weight: bold;
-  padding-bottom: 2px;
-  padding-top: 2px;
-  user-select: none;
-}
-
-.top-bar__apply-icon {
-  display: inline-block;
-  line-height: 12px;
-}
-
-.top-bar__dropdown-arrow {
```

Comment:
Dunno where this was used in the past but, shrug, not seeing any uses of it now.